<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calib420mA</class>
 <widget class="QFrame" name="Calib420mA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>403</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Arial&quot;;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lb420mA1">
      <property name="styleSheet">
       <string notr="true">border: 2px  solid  rgb(0, 170, 255);</string>
      </property>
      <property name="text">
       <string>y=Ax+B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb420mA2">
      <property name="styleSheet">
       <string notr="true">border: 2px  solid  rgb(0, 170, 255);</string>
      </property>
      <property name="text">
       <string>y=Ax+B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb420mA3">
      <property name="styleSheet">
       <string notr="true">border: 2px  solid  rgb(0, 170, 255);</string>
      </property>
      <property name="text">
       <string>y=Ax+B</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb420mACar">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>127</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>标定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLineEdit" name="le5mA"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="le15mA"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pb5mA">
      <property name="text">
       <string>5mA</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="le10mA"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pb10mA">
      <property name="text">
       <string>10mA</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pb15mA">
      <property name="text">
       <string>15mA</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="le19mA"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pb19mA">
      <property name="text">
       <string>19mA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_49">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ff0000;&quot;&gt;*注：数采仪上对应模拟口上下限&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ff0000;&quot;&gt;设定为4.00-20.00方可进行标定。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
